docs/user_guide/assessment/error_estimation.rst

```diff
@@ -0,0 +1,360 @@
+.. _error_estimation:
+
+Error Estimation
+================
+
+.. currentmodule:: fairlearn.metrics
+
+It is a fact universally acknowledged that a person in possession
+of a metric must be in want of an error bar.
+Indeed, the omnipresence of random noise means that anything purporting
+to represent the real world which does not feature an accompanying
+error estimate should be regarded with suspicion.
+When performing a fairness analysis this concern becomes acute,
+since we divide our sample into smaller groups, increasing
+the relative effects of random noise.
+We are then generally interested in the difference or ratio
+of function evaluations on these groups, and errors always
+accumulate, even when the target values (the difference or ratio
+in this case) are getting smaller.
+:ref:`Intersecting groups <intersecting_groups>` make the
+problem worse, since some intersections can have very low
+sample counts, or even be empty.
+
+In Fairlearn, we offer bootstrapping as a means of estimating
+errors.
```

Comment:
The language needs to be more precise here. Many of the metrics reported in metric frame are estimates of errors (like FPR, FNR, classification error, mean square error, etc.)--the bootstrap is used to estimate confidence intervals of these metrics.